### ISSUE TITLE
[BUG] YouTube video section issues

### ISSUE THREAD

**Issue by Hawkedon** [jurialmunkey/skin.arctic.horizon.2#442]
### Skin section

Video Info Dialog

### Current Behavior

The YouTube video section may list unrelated items due to the name conflict. For example, for movie "Monster Hunter (2020)", the video section lists games, etc. This issue can be solved by appending "$INFO[Item.Year] Movie" to the search.  Other items (TV Show/Season/Episode/Music/Album/Song) can be solved in the similar way.

The "Next Page" item in the YouTube video section should be hid due to the non-functional.

### Expected Behavior

As suggestion above.

### Steps To Reproduce

Please check the YouTube video section for "Monster Hunter (2020)" in video info dialog.

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Might be a problem for episodes because the year will be different from the show.

**Comment by Hawkedon** [comment]
Hi @jurialmunkey, I used something like the following before, it works very well for me:

```xml
    <variable name="Info_Youtube_Search">
        <value condition="String.IsEqual(ListItem.DBType,set) | ListItem.IsCollection">$INFO[ListItem.Set]</value>
        <value condition="String.IsEqual(ListItem.DBType,movie)">$INFO[ListItem.Title] $INFO[ListItem.Year] Movie</value>
        <value condition="String.IsEqual(ListItem.DBType,tvshow)">$INFO[ListItem.TVShowTitle] $INFO[ListItem.Year] TV Show</value>
        <value condition="String.IsEqual(ListItem.DBType,season)">$INFO[ListItem.TVShowTitle] $INFO[ListItem.Year] Season $INFO[ListItem.Season]</value>
        <value condition="String.IsEqual(ListItem.DBType,episode)">$INFO[ListItem.TVShowTitle] $INFO[ListItem.Year] Season $INFO[ListItem.Season] Episode $INFO[ListItem.Episode] $INFO[ListItem.Title]</value>
        <value condition="[String.IsEqual(ListItem.DBType,actor) | String.IsEqual(ListItem.Property(item.type),person)]">$INFO[ListItem.Label] Actor</value>
        <value condition="String.IsEqual(ListItem.DBType,artist)">$INFO[ListItem.Artist] Artist</value>
        <value condition="String.IsEqual(ListItem.DBType,album)">$INFO[ListItem.AlbumArtist] $INFO[ListItem.Album] $INFO[ListItem.Year]</value>
        <value condition="String.IsEqual(ListItem.DBType,song)">$INFO[ListItem.Artist] $INFO[ListItem.Title]</value>
        <value>$INFO[ListItem.Label]</value>
    </variable>

```

**Comment by jurialmunkey** [comment]
Okay I've adapted your variable a little bit. I think these should give some good results.

Only one that is a bit of a pain is tvshows. I found with the year and "TV Show" affix it was missing a lot of results (e.g. with Breaking Bad the first results coming up were talk show appearances). Not sure what to do there. Searching a season is good enough with "Season X" added. And Episodes work well with the episode name rather than numbers.

For Artists / Albums etc. I found I got better results with a generic "Music" rather than specific "Artist" "Album" etc. -- not in the actual music info dialog yet but will be useful when I add it so just put in the variable now.

**Comment by ShibajiCh** [comment]
Hi @jurialmunkey,

After the latest update the YouTube video section is not appearing anymore under Video Info for both Movies and TV Shows (see screenshots below). Shall I raise a new issue ticket for this? Kindly advise.

![IMG20220925014505](https://user-images.githubusercontent.com/65297828/192117146-963d0a2e-c1a7-4200-b5e9-f0c96991125b.jpg)

![IMG20220925014403](https://user-images.githubusercontent.com/65297828/192117191-589cfcf2-792a-46a4-ad72-153660229608.jpg)

Thanks and Regards,
ShibajiCh.

**Comment by ShibajiCh** [comment]
Hi @jurialmunkey,

Also, what's the last blank item supposed to be in the below screenshot? It's appearing for all Movies and TV Shows under video info. Shall I create a separate issue ticket for this as well? Please advise.

![IMG20220925015341](https://user-images.githubusercontent.com/65297828/192117650-8a20c35c-2d17-480a-a22c-4ef4fe4fd40a.jpg)

Thanks and Regards,
ShibajiCh.

**Comment by jurialmunkey** [comment]
@ShibajiCh - Oops! I had taken Youtube out temporarily to test something and forgot to add it back in before I pushed an update. Fixed now.

The blank item on the end of the keywords widget is to prevent it from hiding if there are no items because then it wouldn't update when you change the focus. I must've messed something up in the visibility condition because it should only show when the list is empty.
